### DISCUSSION TITLE
Question about multiple inputs on Blackhole

### DISCUSSION THREAD

**Discussion by ghost** [ExistentialAudio/BlackHole#319] [Q&A]
Hello,
I'm a guitarist using BlackHole to capture the sound of my guitar via GarageBand, and using it on video calls on Zoom. That works very well. Now, how, could I have Zoom capture at the same time another source like Spotify or iReal Pro (the play-a-long app), so to be able to play on top of some backing tracks or other musical examples, during the call ? Could somebody indicate what the configuration would be?
Thank you!
All the best

**Comment by devinroth** [comment]
Use both 16ch and 2ch. One for input the other for output.

**Reply by ghost** [reply]
Thanks very much for your reply. 
Would it be possible to tell the exact steps I should follow? Should I install the 2ch version too, and then use it together with the 16ch version ad the output? Right now, in Garageband, I have the BlackHole 16ch set as an input and a Multi-Output Device set as the output. In Zoom, the input is set to BlackHole 16ch and the output set to my Presonus Audiobox 96 audio interface.
All the best and thanks again for you help.

**Reply by high-solutions** [reply]
instal the 2ch version, use this as the output instance. use the 16ch as input on garage band (and maybe other apps but not on zoom), make your apps (garage band and everything else) sending the output to the 2 channel, in zoom select the 2ch as mic and leave the audiobox as output. If your other apps don't have an output patch you need to select the 2ch as output device in your mac - midi/audio settings. (and you can make an multi output device with your 2ch and headphones jack or what ever to make a monitor device, if you do this select this new multi output as standard speaker device) hope it's clear

**Reply by ghost** [reply]
Thank you very much! I'll give it a try. Have a nice day
